ArchiSteamFarm/Commands.cs

```diff
@@ -1169,6 +1173,47 @@ internal sealed class Commands {
 			return responses.Count > 0 ? string.Join(Environment.NewLine, responses) : null;
 		}
 
+		private async Task<string> ResponseLevel(ulong steamID) {
+			if (steamID == 0) {
+				Bot.ArchiLogger.LogNullError(nameof(steamID));
+				return null;
+			}
+
+			if (!Bot.IsConnectedAndLoggedOn) {
+				return FormatBotResponse(Strings.BotNotConnected);
+			}
+
+			if (!Bot.IsMaster(steamID)) {
+				return null;
+			}
+
+			uint? level = await Bot.ArchiHandler.GetLevel().ConfigureAwait(false);
+
+			if (level == null) {
+				return FormatBotResponse(Strings.WarningFailed);
+			}
+
+			return FormatBotResponse(string.Format(Strings.BotLevel, level));
+		}
+
+		private static async Task<string> ResponseLevel(ulong steamID, string botNames) {
+			if ((steamID == 0) || string.IsNullOrEmpty(botNames)) {
+				ASF.ArchiLogger.LogNullError(nameof(steamID) + " || " + nameof(botNames));
+				return null;
+			}
+
+			HashSet<Bot> bots = Bot.GetBots(botNames);
+			if ((bots == null) || (bots.Count == 0)) {
+				return ASF.IsOwner(steamID) ? FormatStaticResponse(string.Format(Strings.BotNotFound, botNames)) : null;
+			}
+
+			IList<string> results = await Utilities.InParallel(bots.Select(bot => Task.Run(() => bot.Commands.ResponseLevel(steamID)))).ConfigureAwait(false);
```

Comment:
Using `Task.Run()` here is inappropriate since target method is already `async Task`, check other calls to async methods.